rxbindings/src/test/java/com/amplifyframework/rx/RxAuthBindingTest.java

```diff
@@ -86,7 +87,7 @@ public void testSignUpSucceeds() {
         AuthCodeDeliveryDetails details = new AuthCodeDeliveryDetails(RandomString.string(), DeliveryMedium.SMS);
         AuthSignUpStep step = AuthSignUpStep.CONFIRM_SIGN_UP_STEP;
         AuthNextSignUpStep nextStep = new AuthNextSignUpStep(step, Collections.emptyMap(), details);
-        AuthSignUpResult result = new AuthSignUpResult(false, nextStep);
+        AuthSignUpResult result = new AuthSignUpResult(false, nextStep, new AuthUser(userId, username));
```

Comment:
In the JavaDoc comment for the AuthSignUpResult constructor, it says the user will be null if isSignUpComplete is false, but here you're passing in a non-null AuthUser and false for isSignUpComplete. When creating this new AuthSignUpResult here, should you either pass in true for isSignUpComplete or null for the user?

Reply:
Good point - I'll add null for the user